docs/rst_man/src/sampler/filesingle/ldms-sampler-filesingle.rst

```diff
@@ -0,0 +1,129 @@
+.. _filesingle:
+
+=================
+filesingle
+=================
+
+:Date:   15 Dec 2018
```

Comment:
I think we need the following in each man page file as well:

:Manual section: 
:Manual group: